integration-tests/http/__tests__/product/admin/product-import.spec.ts

```diff
@@ -454,7 +505,7 @@ medusaIntegrationTestRunner({
         )
       })
 
-      it("should fail on invalid region in prices being present in the CSV", async () => {
+      it.skip("should fail on invalid region in prices being present in the CSV", async () => {
```

Comment:
Currently we fetch and test the imported file for the region ids.

However, in the revamp, I have moved all DB calls to the processing of the CSV file. Therefore we cannot verify for valid regions until the file is processed.

Should I remove this test and add a new one once the processing of CSV is also revamped?